lib/Conversion/SeqToSV/SeqToSV.cpp

```diff
@@ -456,48 +456,68 @@ void SeqToSVPass::runOnOperation() {
   if (failed(applyPartialConversion(circuit, target, std::move(patterns))))
     signalPassFailure();
 
+  bool needsMemRandomization = !memsByModule.empty();
+
+  auto loc = UnknownLoc::get(context);
+  auto b = ImplicitLocOpBuilder::atBlockBegin(loc, circuit.getBody());
+  if (needsRegRandomization || needsMemRandomization) {
+    b.create<sv::MacroDeclOp>("ENABLE_INITIAL_REG_");
```

Comment:
Why are these declared without checking if a previous pass declared them (like below)?  Are these /only/ for code here to rely on?

Reply:
(and an error if any other pass declares them?)

Reply:
Only one can be declared, so if they are redefined, we get a double-declaration error. I don't want to add that many more checks since I'd like to re-do this whole mechanism very soon.